gammapy/spectrum/models.py

```diff
@@ -337,6 +352,28 @@ def evaluate(energy, index, amplitude, reference, lambda_):
             cutoff = exp(-energy * lambda_)
         return pwl * cutoff
 
+    def to_sherpa(self, name='default'):
+        """Return `~sherpa.models.Arithmetic model`
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the sherpa model instance
+        """
+        # NOTE: we cannot use naima.sherpa_models.SherpaModelECPL since it is
+        # meant to be used as abstract base class (Arithmetic model only
+        # initialized in daughter classes)
+        # see https://github.com/zblz/naima/blob/master/naima/sherpa_models.py#L149
```

Comment:
I don't think that line number is correct any more.

It should be this, no?
https://github.com/zblz/naima/blob/8ab6a69a9e95da7baed9c33cb9dde56de0309b69/naima/sherpa_models.py#L74

My suggestion would be to remove the note.
We'll heavily develop spectral and spatial model classes in the coming months and this historical note of what's in naima won't be useful I think.